client/src/components/Onboarding/OnboardingUser.scss

```diff
@@ -0,0 +1,42 @@
+.onboarding {
+  padding: 1em 1em 1em 1em;
+}
```

Comment:
__Inconsistent & Inaccurate__: This looks like quickie spacing values that do not match other sections. If it is quickie code, then please:

1. Add the [`client/src/styles/settings/space.css` file](https://github.com/TACC/Frontera-Portal/pull/236/files#diff-985014982b37ef957ccafe9ccfd32f8f41ab8314ee1d87db155603ceefa880b1) from #236.
2. [Load the global styles in `client/src/styles/global.css`](https://github.com/TACC/Frontera-Portal/pull/236/files#diff-092e0cc8b1bb33efceb13b039b894e9bdf2a3c2e260da906da0f20c1c069341bR17) (like in #236).
3. Use the `--global-space--section` variable (i.e. `padding: var(--global-space--section);`).

Reply:
I don't understand your instructions. Do you have example code where this is being used? The Load the global styles in client/src/styles/global.css link goes to the top of the PR rather than a line number.

Reply:
Yes, [similar usage example](https://github.com/TACC/Frontera-Portal/pull/236/files#diff-bbc8bdd1df06b65b9735e81cf12b7a3c4ce9bfdc10a7a5d52f77d7c142448d23) exists (added link and footnote in original comment); but, I've written out the usage code for this case. And, I've fixed the link and noted the relevant line number of step 2. (The referenced PR is so large, that sometimes browser cannot scroll to file and line, because file is not loaded.)

Reply:
Ahh okay you want me to actually bring in the file? space.css is not in master yet.

Reply:
Yes. Objections and concerns welcome.

_This request can reasonably be ignored, until this PR is merged into `master`; after which, I will update `OnboardingUser` to use `Section[…]` components (in new commit to #236)._